_source/_posts/2022-02-21-build-deploy-node-app-heroku.md

```diff
@@ -0,0 +1,335 @@
+---
+layout: blog_post
+title: "Build and Deploy a Node.js App to Heroku"
+author: nickolas-fisher
+by: contractor
+communities: [javascript]
+description: "A tutorial that shows you how to build and deploy a Node.js app to Heroku."
+tags: [javascript, node, heroku, express]
+tweets:
+- "Learn how to build and deploy a @nodejs app to @heroku 🔥"
+image: blog/build-deploy-node-app-heroku/cover-image.jpg
+type: conversion
+github: https://github.com/oktadev/okta-node-heroku-app-example
+---
+
+Heroku is a platform as a service that supports many languages.  Initially, it supported only Ruby sites but now supports various languages, including JavaScript with Node.js.  Heroku also has docker support so that you can deploy just about anything to it.  
```

Comment:
```suggestion
Heroku is a platform as a service (PaaS) that supports many languages.  Initially, it supported only Ruby sites but now supports various languages, including JavaScript with Node.js.  Heroku also has Docker support so that you can deploy just about anything to it.  
```
Capitalize Docker b/c it's a product name